### ISSUE TITLE
[Bug] Error to add TXT record DNS (Error: invalid restart format :: false)

### ISSUE THREAD

**Issue by grupoactialia** [hestiacp/hestiacp#2557]
### Describe the bug

I try to add a TXT record dns and I have a error "Error: invalid restart format :: false"

![Captura de pantalla 2022-04-28 a las 8 06 49](https://user-images.githubusercontent.com/33631434/165687827-106e46c5-761a-4e09-a48c-45ece0196764.png)

### Tell us how to replicate the bug

1. Click on DNS tab. 
2. Click in a one website 
3. Add record 
4. Set type TXT record 
5. Save

### Which components are affected by this bug?

Control Panel Backend, Control Panel Web Interface

### Hestia Control Panel Version

1.5.12

### Operating system

Ubuntu 20.04

### Log capture

```shell
/var/log/hestia/error.log

2022-04-28 07:42:11 v-add-dns-record  'admin' 'domain.es' '@' 'TXT' 'txtvalue' '' '' 'false' '' [Error 2]
2022-04-28 07:48:17 v-add-dns-record  'admin' 'domain.es' '@' 'TXT' 'txtvalue' '' '' 'false' '' [Error 2]
```

**Comment by jaapmarcus** [comment]
https://github.com/hestiacp/hestiacp/blob/48dbab4bd2ae947bc1d0fc9b13a40a07f2a89110/web/add/dns/index.php#L172

**Comment by jaapmarcus** [comment]
Issue has been patched 1.5.13 will follow soon
